src/components/SetupOpportunity/SetupOpportunity.tsx

```diff
@@ -23,18 +23,18 @@ export const SetupOpportunity: FC<Props> = ({ opportunity }) => {
 
     const linkToFunders = `/setup/${opportunity.getOpportunity.id}/funders`;
 
-    const hasWebsiteListings =
-        opportunity.getOpportunity &&
+    const websiteListings =
         opportunity.getOpportunity.websiteListings &&
-        opportunity.getOpportunity.websiteListings.items;
+        opportunity.getOpportunity.websiteListings.items
+            ? opportunity.getOpportunity.websiteListings.items
+            : undefined;
 
-    const hasApplications =
+    const applications =
         opportunity.getOpportunity.application &&
-        opportunity.getOpportunity.application.items;
+        opportunity.getOpportunity.application.items
```

Comment:
No need for the ternary here: just do `|| null`

Reply:
fixed